x-pack/test/security_api_integration/packages/helpers/saml/idp_metadata_mock_idp.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata"
+                     entityID="urn:mock-idp">
```

Comment:
It is a temp solution, ideally we need to generate metadata file dynamically (like for serverless)